### PR TITLE
Publisher page is v2

### PR DESCRIPTION
**Pull request by jdlrobson**
* Descope the carousel a la subjects page
and migrate it to slick (without facets -
see f534362)
* Remove some inline styles

Fixes: #1554

Closes #<IssueNumber>

### CODE REVIEW

File: openlibrary/templates/publishers/view.html [hunk 1]
```diff
@@ -39,15 +39,8 @@ <h1>
...
-<script type="text/javascript">
-<!--
-var page = new Subject($:json_encode(page));
-//-->
-</script>
-
-$:render_template("lib/covers", page)
-
-<div id="contentMeta" style="padding-top:0;">
+<div class="contentBody" style="padding-top:0;">
```
Comment (mekarpeles) [inline openlibrary/templates/publishers/view.html]: any way we can remove the inline style?

File: openlibrary/templates/publishers/view.html [hunk 2]
```diff
@@ -39,15 +39,8 @@ <h1>
...
+    $:render_template("books/custom_carousel", books=page.works, key="subjects")
 
 	<div class="head">
 	    <h2 style="margin-bottom:0px!important;">
```
Comment (mekarpeles) [inline openlibrary/templates/publishers/view.html]: can we remove this inline style and preferably also the `!important`?

File: openlibrary/templates/publishers/view.html [hunk 3]
```diff
@@ -482,11 +229,11 @@ <h2 style="margin-bottom:0px!important;">
 //-->
 </script>
 
-    <div class="chart" style="width:898px;height:140px;">
```
Comment (mekarpeles) [inline openlibrary/templates/publishers/view.html]: ty!

**Comment by jdlrobson** [comment]
Done

**Comment by jdlrobson** [comment]
@mekarpeles were those the changes you needed?

**Comment by mekarpeles** [comment]
yes thank you! lgtm

**Pull request merged**